multiversx_sdk/core/transactions_factories/relayed_transactions_factory_test.py

```diff
@@ -231,3 +232,76 @@ def test_compute_relayed_v2_transaction(self):
         assert relayed_transaction.options == 0
         assert relayed_transaction.gas_limit == 60414500
         assert relayed_transaction.data.decode() == "relayedTxV2@000000000000000000010000000000000000000000000000000000000002ffff@0f@676574436f6e7472616374436f6e666967@fc3ed87a51ee659f937c1a1ed11c1ae677e99629fae9cc289461f033e6514d1a8cfad1144ae9c1b70f28554d196bd6ba1604240c1c1dc19c959e96c1c3b62d0c"
+
+    def test_compute_relayed_v3_transaction(self):
+        alice = self.wallets["alice"]
+        bob = self.wallets["bob"]
+
+        inner_transaction_1 = Transaction(
+            sender=bob.label,
+            receiver=bob.label,
+            gas_limit=50000,
+            chain_id="T",
+            nonce=0,
+            version=2,
+            relayer=alice.label
+        )
+
+        inner_transactions = [inner_transaction_1]
+        serialized_inner_transaction_1 = self.transaction_computer.compute_bytes_for_signing(inner_transaction_1)
+        inner_transaction_1.signature = bob.secret_key.sign(serialized_inner_transaction_1)
+
+        relayed_transaction = self.factory.create_relayed_v3_transaction(
+            relayer_address=Address.from_bech32(alice.label),
+            inner_transactions=inner_transactions
+        )
+        serialized_relayed_transaction = self.transaction_computer.compute_bytes_for_signing(relayed_transaction, with_signature=True)
```

Comment:
you should not serialize a transaction to be signed with the signature field, but the the `with_signature` param should not be present on `compute_bytes_for_signing`

Reply:
done :+1: